src/drawing.cpp

```diff
@@ -148,7 +148,7 @@ void MatrixPreDraw()
 
 // WiFiDraw
 //
-// Draws forom WiFi color data if available, returns pixels drawn this frame
+// Draws from WiFi color data if available, returns pixels drawn from this frame

```

Comment:
The second "from" is correctly absent in the original sentence. The function returns the pixels drawn while composing the current frame, so "drawn this frame".

```suggestion
// Draws from WiFi color data if available, returns pixels drawn this frame
```